### ISSUE TITLE
2.3: Userlist UX/accessibility issue

### ISSUE THREAD

**Issue by PhMemmel** [bigbluebutton/bigbluebutton#11839]
**Describe the bug**
Trying to move through the user list with arrow down/up key is quite difficult/not possible, because every second user is being left out.

**To Reproduce**
1. Open meeting in 2.3-beta1
2. Let some users join
3. tab to user list (or focus with clicking on a user and closing popup menu again)
4. trying to navigate with down/up arrow keys: only every second user is accessible.

**Expected behavior**
Every user can be selected by arrow keys, arrow down key selects **next** user

**Actual behavior**
Every second user gets left out. In case of an even number of users it's impossible to select half of the users.
Additionally: When the list is focused by using tab key, first user should be selected. At the moment you have to use "arrow down key" and "arrow up key" to select own user (first user).

**Screenshots**

https://user-images.githubusercontent.com/65113153/113275972-a34c9700-92df-11eb-8a9f-a0e877bc92b5.mp4

I pressed "down arrow key" once every second in the clip.

**BBB version (optional):**
tested on 2.3-beta1

**Desktop:**
tested on current Chrome

**Additional context**

- IMHO scrolling through the user list with arrow down key should stop at last user instead of starting on top again.
- It would also be nice being able to use "Pos1" and "End" key to move to the beginning/end of the user list. It would make the list more clear to use. This might also improve accessibility of the user list in general. I'm no accessibility expert though.

**Comment by PlumedArcher** [comment]
This also happens on Firefox. When pressing down it always starts with the second user, regardless of which name you click first. Pressing up starts with the second last user, as if the default position was 0 on the list. After that the navigation behaves like discribed above.
